Lib/test/test_tkinter/test_misc.py

```diff
@@ -491,6 +491,8 @@ def test2(e): pass
         f.unbind(event, funcid)
         script = f.bind(event)
         self.assertNotIn(funcid, script)
+        self.assertIn(funcid2, script)
```

Comment:
To properly test the unbind code (see comment), I think this should bind 3 scripts and unbind the middle one.